src/plugins/intel_gna/tests/functional/pass_tests/conv_with_padding.cpp

```diff
@@ -120,38 +120,52 @@ const std::vector<std::map<std::string, std::string>> configs_gna_3_0 = {
 const std::vector<std::map<std::string, std::string>> configs_gna_3_5 = {
     {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_5"}}};
 
-const std::vector<size_t> input = {1, 8, 16, 16};
-const std::vector<size_t> filter = {8, 8, 2, 2};
-const std::vector<std::ptrdiff_t> no_padding{};
-const std::vector<std::ptrdiff_t> padding{1, 1};
+const std::vector<size_t> input2D = {1, 8, 16, 16};
+const std::vector<size_t> filter2D = {8, 8, 2, 2};
+const std::vector<std::vector<size_t>> inputs2D_gna_3_5 = {{1, 1, 2, 16}, {1, 1, 16, 16}};
+const std::vector<std::vector<size_t>> inputs1D_gna_3_5 = {{1, 1, 1, 16}};
+const std::vector<std::vector<size_t>> filters1D_gna_3_5 = {{1, 1, 1, 2}, {1, 1, 1, 16}};
+const std::vector<std::ptrdiff_t> no_padding ={0, 0};
+const std::vector<std::ptrdiff_t> padding1D = {0, 1};
+const std::vector<std::ptrdiff_t> padding2D = {1, 1};
 
 INSTANTIATE_TEST_SUITE_P(smoke_conv_without_padding,
                          ConvWithPaddingTestPos,
                          ::testing::Combine(::testing::Values(net_precisions),
                                             ::testing::Values(CommonTestUtils::DEVICE_GNA),
                                             ::testing::ValuesIn(configs_gna_3_0_to_3_5),
-                                            ::testing::Values(input),
-                                            ::testing::Values(filter),
+                                            ::testing::Values(input2D),
+                                            ::testing::Values(filter2D),
                                             ::testing::Values(no_padding)),
                          ConvWithPaddingTestPos::getTestCaseName);
 
-INSTANTIATE_TEST_SUITE_P(smoke_conv_with_padding_gna_3_5,
+INSTANTIATE_TEST_SUITE_P(smoke_conv_with_input1D_filter1D_padding_gna_3_5,
                          ConvWithPaddingTestPos,
                          ::testing::Combine(::testing::Values(net_precisions),
                                             ::testing::Values(CommonTestUtils::DEVICE_GNA),
                                             ::testing::ValuesIn(configs_gna_3_5),
-                                            ::testing::Values(input),
-                                            ::testing::Values(filter),
-                                            ::testing::Values(padding)),
+                                            ::testing::ValuesIn(inputs1D_gna_3_5),
+                                            ::testing::ValuesIn(filters1D_gna_3_5),
+                                            ::testing::Values(padding1D)),
+                         ConvWithPaddingTestPos::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_conv_with_input2D_filter1D_padding_gna_3_5,
```

Comment:
General comment: we're adding a missing path for 1D convolutions validator, but test cases are added only in padding tests. I would see more tests for 1D path in existing tests for convolutions without padding. Besides, maybe it's worth to add some combinations of input/kernel/padding sizes, maybe with pooling as well, to make sure all code paths are covered.

Reply:
+ enabling 1D convolution testing for target 3.5 in smoke_1DPaddedToValid test suite
+ smoke_Decompose2DConvGNA35
+ test which validates that mapping from 2D to 1D convolution does not take place for target 3.5